### PR TITLE
add riscv64 support

### PR DESCRIPTION
**Pull request by gdams**
Currently RISC-V is not supported in this action and it would be useful for testing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,7 +152,8 @@ This table details the valid `arch`/`distro` combinations:
 | armv6    | jessie, stretch, buster, bullseye, alpine_latest |
 | armv7    | jessie, stretch, buster, bullseye, ubuntu16.04, ubuntu18.04, ubuntu20.04, ubuntu22.04, ubuntu_latest, ubuntu_rolling, ubuntu_devel, fedora_latest, alpine_latest, archarm_latest |
 | aarch64  | stretch, buster, bullseye, ubuntu16.04, ubuntu18.04, ubuntu20.04, ubuntu22.04, ubuntu_latest, ubuntu_rolling, ubuntu_devel, fedora_latest, alpine_latest, archarm_latest |
-| s390x    | jessie, stretch, buster, bullseye, ubuntu16.04, ubuntu18.04, ubuntu20.04, ubuntu22.04, ubuntu_latest, ubuntu_rolling, ubuntu_devel, fedora_latest, alpine_latest |
+| riscv64    | jessie, stretch, buster, bullseye, ubuntu20.04, ubuntu22.04, ubuntu_latest, ubuntu_rolling, ubuntu_devel, fedora_latest, alpine_latest |
```
Comment (martin-g) [inline README.md:155]:
```suggestion
| riscv64  | jessie, stretch, buster, bullseye, ubuntu20.04, ubuntu22.04, ubuntu_latest, ubuntu_rolling, ubuntu_devel, fedora_latest, alpine_latest |
```

File: README.md [hunk 2]
```diff
@@ -152,7 +152,8 @@ This table details the valid `arch`/`distro` combinations:
 | armv6    | jessie, stretch, buster, bullseye, alpine_latest |
 | armv7    | jessie, stretch, buster, bullseye, ubuntu16.04, ubuntu18.04, ubuntu20.04, ubuntu22.04, ubuntu_latest, ubuntu_rolling, ubuntu_devel, fedora_latest, alpine_latest, archarm_latest |
 | aarch64  | stretch, buster, bullseye, ubuntu16.04, ubuntu18.04, ubuntu20.04, ubuntu22.04, ubuntu_latest, ubuntu_rolling, ubuntu_devel, fedora_latest, alpine_latest, archarm_latest |
-| s390x    | jessie, stretch, buster, bullseye, ubuntu16.04, ubuntu18.04, ubuntu20.04, ubuntu22.04, ubuntu_latest, ubuntu_rolling, ubuntu_devel, fedora_latest, alpine_latest |
+| riscv64  | jessie, stretch, buster, bullseye, ubuntu20.04, ubuntu22.04, ubuntu_latest, ubuntu_rolling, ubuntu_devel, fedora_latest, alpine_latest |
```
Comment (martin-g) [inline README.md:155]: You need to add Dockerfiles for the Debian and Fedora versions.

Comment (gdams) [reply README.md]: that was a typo (there is currently no stable debian support so I've updated the README

**Comment by uraimo** [comment]
Hi @gdams , please let me know when you are happy with this, I planned to add multiple new archs along with a refactoring of the Dockerfile mechanism but I'm ok with adding platforms with at least one working distribution even before that. Like in this case.

**Comment by gdams** [comment]
This should be good to go @uraimo (not sure why the armv7l job is failing but it's unrelated to my changes

**Comment by uraimo** [comment]
Yeah completely unrelated, there is a known issue with Fedora running inside QEMU on some platforms #86.

**Pull request merged**